lib/gooddata/lcm/helpers/check_helper.rb

```diff
@@ -27,6 +27,7 @@ def check_params(specification, params)
               end
 
               if specification[param_name][:opts][:deprecated]
+                params[specification[param_name][:opts][:replacement]] = value
```

Comment:
Refers to 'specification' more than self (maybe move it to another class?) (FeatureEnvy)